worlds/stardew_valley/strings/craftable_names.py

```diff
@@ -174,9 +175,11 @@ class ModEdible:
 
 class ModCraftable:
     travel_core = "Travel Core"
-    glass_bazier = "Glass Bazier"
+    glass_bazier = "Glass Brazier"

```

Comment:
Should the typo in the constant name be fixed as well?

Reply:
Yeah they finally fixed it